server/src/utils.ts

```diff
@@ -203,4 +203,13 @@ export function nLog(message: string, settings: NavigatorSettings){
     }
 }
 
+export function getPerlimportsProfile (settings: NavigatorSettings): string[] {
+    let profileCmd: string[] = [];
+    if (settings.perlimportsProfile) {
+        let profile = settings.perlimportsProfile;
```

Comment:
You could `const` here, too. Or better yet, don't create the variable at all. Also, `.push()` will take multiple args.

So you could replace the contents of this block with:

```ts
profileCmd.push('--config-file', settings.perlimportsProfile);
```